vmdb/db/fixtures/ae_datastore/ManageIQ/Cloud/Orchestration/Provisioning/StateMachines/Methods.class/__methods__/provision.rb

```diff
@@ -7,8 +7,10 @@
 service = $evm.root["service_template_provision_task"].destination
 
 begin
-  service.deploy_orchestration_stack
+  ems_ref = service.deploy_orchestration_stack
+  $evm.log("info", "Stack #{service.stack_name} with reference id (#{ems_ref}) is being created")
```

Comment:
Good to learn. Here `ems_ref` cannot be `nil`, otherwise an exception would have been thrown.